integration-tests/tests/run-workflow-on-snapshot.js

```diff
@@ -0,0 +1,83 @@
+const _ = require('lodash/fp')
+const pRetry = require('p-retry')
+const { withWorkspace } = require('../utils/integration-helpers')
+const { click, clickable, delay, dismissNotifications, findElement, findText, fillIn, input, select, signIntoTerra, waitForNoSpinners, findInGrid, navChild } = require('../utils/integration-utils')
+const { withUserToken } = require('../utils/terra-sa-utils')
+
+
+const snapshotName = 'testsnapshot'
+const findWorkflowButton = clickable({ textContains: 'Find a Workflow' })
+
+const testRunWorkflowOnSnapshotFn = _.flow(
+  withWorkspace,
+  withUserToken
+)(async ({ dataRepoUrl, environment, page, testUrl, snapshotColumnName, snapshotId, snapshotTableName, token, workflowName, workspaceName }) => {
```

Comment:
Instead of providing the data repo url, we should pull it from the config, like we do in `import-dockstore-workflow`:

```js
  const { dataRepoUrlRoot } = await fetch(`${testUrl}/config.json`).then(res => res.json())
```